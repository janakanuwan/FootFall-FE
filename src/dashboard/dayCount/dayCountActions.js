import { createAction } from 'redux-actions';
import { SET_DAY_COUNT } from "../../const/action-types";

const setDayCount = createAction(
  SET_DAY_COUNT,
  ({ dayName, date, count }) => ({ dayName, date, count })
);

export { setDayCount };
