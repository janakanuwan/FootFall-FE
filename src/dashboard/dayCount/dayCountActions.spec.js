import { setDayCount } from "./dayCountActions";

describe('dayCountActions', () => {
  it('should create an action for set day count', () => {
    expect(setDayCount({ dayName: 'Today', date: '2018-04-01', count: 0 })).toMatchSnapshot();
  });
});
