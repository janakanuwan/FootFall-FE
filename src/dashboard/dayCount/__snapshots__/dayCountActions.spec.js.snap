// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dayCountActions should create an action for set day count 1`] = `
Object {
  "payload": Object {
    "count": 0,
    "date": "2018-04-01",
    "dayName": "Today",
  },
  "type": "SET_DAY_COUNT",
}
`;
